oak-authorization-principalbased/src/main/java/org/apache/jackrabbit/oak/spi/security/authorization/principalbased/impl/AbstractEntry.java

```diff
@@ -16,7 +16,7 @@
  */
 package org.apache.jackrabbit.oak.spi.security.authorization.principalbased.impl;
 
-import com.google.common.base.Objects;
+import org.apache.jackrabbit.guava.common.base.Objects;
```

Comment:
in other PR this was replaced by MoreObjects, no?

Reply:
It's a specific method in "Objects" that moved to "MoreObjects". Others stayed where they were.